test/clean.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+docker images -q 'librarytest/*' | xargs docker rmi -f
```

Comment:
Ah, good point; there is also a possibility (however slim) that one of our
librarytest images is the same layer as another legitimate image that's not
ours, especially since the Dockerfiles will usually be really simplistic.
:innocent: